examples/edf-methanesat-ee/projects_edf-methanesat-ee_assets_public-preview_L3concentration.js

```diff
@@ -0,0 +1,11 @@
+var dataset = ee.ImageCollection("projects/edf-methanesat-ee/assets/public-preview/L3concentration");
```

Comment:
You might want to add filterDate(). As you add more images, displaying the whole collection will be slower and slower. Ditto elsewhere.

Reply:
Thanks, done.  Added to L3concentration and L4area.  I didn't add it to L4point but will add a TODO for myself to see how to filter the table in the future.